### DISCUSSION TITLE
obtaining money with the menu

### DISCUSSION THREAD

**Discussion by younes569** [YimMenu/YimMenu#1460] [Q&A]
Is there any way I could get money from this menu like you can change in the stat editor (just like the rp example "$MPX_CHAR_SET_RP_GIFT_ADMIN"0. Or is there a list with more  of these commands?

**Comment by BigboiKilljoy** [comment]
No getting money with the menu because they dont want Take-Two being angry about a loss in their precious shark card sales

**Comment by BigboiKilljoy** [comment]
Just bounty everyone and commit mass murder, preferably with out god mode.

**Reply by younes569** [reply]
what about a script that could rig the slot machines like in the kiddions menu

**Reply by FIying-Scotsman** [reply]
> what about a script that could rig the slot machines like in the kiddions menu

You have to remember we're balancing on a fine line as while T2 target P2Cs and other paid content (for obvious reasons), they've targeted free mods in the past (i.e OpenIV, the GTA IV map conversion, re3/reVC etc etc). Adding money/recovery options like that will very quickly get more unwanted attention than this project already gets.

No one is stopping you from doing it in private, but it will never be an official script/feature.

**Comment by SlavicBlin** [comment]
There is a really slow an not really good way of makin 2k every now and then and thats by spamming the time warp button on max, the game will then give you 2k for good behaviour but this is really slow and probably detectable 👍
